# Message containing information about horizontal location of
# an object in an image
bool contains_object
float64 horizontal_relative_position